pages/automatedtesting.md

```diff
@@ -1,39 +1,146 @@
+1. **Manual Testing & Code Review**- slow and time consuming
+2. **In the wild** - End users encounter bugs while trying to use the software - not good.
+3. **Automated Testing** - Initial setup can be difficult, but saves time in the long run.
 
-It focused on writing great issues for bugs, mistakes, and other problems that may be found in the codebase. It explained that issues should be constructive, rather than negative, which is to say that rather than just saying "This doesn't work," it is much more appropriate to describe in detail the problem encountered, even including screenshots, when they may be helpful.   
-But, even more importantly, it means that one should take a proactive stance and try to figure out how a problem may be addressed, so that the issues end up looking much more like proposals for improvement than complaints about stuff not working properly.
 
-Then, the project focused on manual tests, and explained how to conduct them and what to expect from them, highlighting pros and cons of such a testing practice. Finally, the project tackled the challenge of introducing automated testing, and envisioned writing Selenium tests in Java.
+Typically manual testing is done by the person developing a new feature, after they write the code they load up the BeLL apps and test the feature they are working on. Code review happens when a pull request is made, and others review your code for bugs and best practices.
 
-The project got abandoned since, but the entire history is still available at [Bug Hunt](bughunt.md).
+The worst way to find bugs is through users - which is what we want to prevent. Releasing updates when not enough testing has been done is just outsourcing the testing to your users, who will not appreciate it. That said, getting select users (interns) to use the software before everyone has to use it can be a good idea. This is called [beta testing](https://en.wikipedia.org/wiki/Software_testing#Beta_testing). A closely related idea called [dogfooding](https://en.wikipedia.org/wiki/Eating_your_own_dog_food) is using your own beta software in the process of doing your work.
 
-## The Present
-More recently, the project was renamed Automated Testing, and was implemented through Selenium, but with tests written in Python, using the `unittest` library.  
-The tests are triggered automatically by Travis CI every time a pull request or a commit is pushed to GitHub (for now automated testing is limited to the [ole-vi repo](https://github.com/ole-vi/BeLL-Apps)).
-What happens is that Travis CI creates an environment in the cloud, which includes a nation, and runs the Selenium tests through a tunnel established with Sauce Labs.
+Automated testing aims to reduce the amount of manual testing needed, and more importantly, reduce the number of bugs found in the wild. There are many kinds of automated testing, but the for this project UI testing is used.
```

Comment:
;)

maybe all of them 

I mostly want to raise more awareness about what the plan is ...
a plan which involves detecting error which could get introduced by every commit
bug which I mostly call regressions

hmm maybe we should call it `regression catching` ;)